v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/options/MongoDbToBigQueryOptions.java

```diff
@@ -158,5 +158,15 @@ public interface JavascriptDocumentTransformerOptions extends PipelineOptions {
     String getJavascriptDocumentTransformFunctionName();
 
     void setJavascriptDocumentTransformFunctionName(String javascriptDocumentTransformFunctionName);
+
+    @TemplateParameter.Boolean(
+        order = 2,
+        optional = true,
+        description = "Use legacy time format.",
+        helpText =
```

Comment:
The description is "use legacy time format" but the option name is "useIsoTimeFormat". The helpText (default is true) is also not true. Please fix.